src/sql/workbench/browser/modal/media/modal.css

```diff
@@ -13,25 +13,137 @@
 	z-index: 500;
 }
 
-.modal:not(.flyout-dialog) .modal-dialog {
+.modal .btn-secondary {
+	border-style: solid;
+	border-width: 1px;
+}
+.modal.callout-dialog .modal-dialog {
+	width: 452px;
+}
+.modal.callout-dialog.compact .modal-dialog {
+	width: 288px;
+}
+
+.modal:not(.flyout-dialog):not(.callout-dialog) .modal-dialog {
 	margin: auto;
 	width: 640px;
 	height: 480px;
 }
 
+.modal.callout-dialog {
+	background-color: transparent;
+}
+.modal.callout-dialog .modal-dialog {
+	border-radius: 2px;
+	box-shadow: 0px 3px 8px rgba(var(--foreground));
+	max-height: 300px;
+	position: absolute;
+}
+
+.modal.callout-dialog .modal-content .insert-image .flex-container {
+	display: flex;
+}
+.modal.callout-dialog .modal-content .insert-image .flex-container > div {
+	flex: 1;
+}
+.modal.callout-dialog .modal-content p {
+	margin: 0;
+}
+.modal.callout-dialog .modal-content .button-icon {
+	cursor: pointer;
+	margin-left: 10px;
+}
+.modal.callout-dialog .modal-content .insert-image .monaco-inputbox {
+	min-width: 380px;
+}
+.modal.callout-dialog .modal-content .row  {
+	margin-bottom: 16px;
+}
+.modal.callout-dialog .modal-content .radio-group input {
+	margin-right: 8px;
+}
+.modal.callout-dialog .modal-content .radio-group span {
+	margin-right: 15px;
+}
+
+.hc-black .modal.callout-dialog .modal-dialog {
+	box-shadow: none;
+}
+
+/* Correct the arrow appearance for HC theme */
+.arrow:before {
```

Comment:
arrow is to generic, lets call it callout-arrow to avoid css collision.

Reply:
Agreed.